practice-two/src/index.html

```diff
@@ -325,7 +325,42 @@ <h3 class="project__item-title">Watch apps</h3>
       <!-- END PRCING SECTION-->
 
       <!-- START CONTACT SECTION -->
-      <section></section>
+      <section class="contact">
+        <div class="container contact__wrapper">
+          <h2 class="heading-section text-color-base">Get in Touch</h2>
+          <form class="contact__form" action="javascript:void(0)">
+            <input type="text" name="name" placeholder="Name*" class="form-input contact__input">
+            <input type="email" name="email" placeholder="Email*" class="form-input contact__input">
+            <textarea name="message" placeholder="Message" class="form-input contact__message"></textarea>
+            <div>
```

Comment:
you can nest the `<input>` directly inside the `<label>` and no need to use `<div>` here. You can refer this [aritcle](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)